docs/src/administration/users.md

```diff
@@ -1,138 +1,162 @@
+---
+title="Using the CLI"
+---
+Use the [Platform.sh CLI (Command Line Interface)](/development/cli/_index.md)to change user permissions.
 
-```bash
-$ platform get <projectID>
+* `platform user:role`
 
-```
+  ```bash
+  platform user:role user1@example.com -r contributor
+  ```
 
-or the `git clone` command visible from the "Git" dropdown in the management console
+Use `platform list` to get the full list of commands.
 
-```bash
-$ git clone git@github.com:user/github-repo.git Project Name
-```
+After you delete a user from a project or an environment type, you must trigger a redeploy to propagate SSH access changes to each environment. You can redeploy by using the CLI command platform `redeploy`.
 
-both would error with
+{{< /codetabs >}}
 
-```bash
-Failed to connect to the Git repository: git@github.com:user/github-repo.git
 
-Please make sure you have the correct access rights and the repository exists.
-```
+### Transfer a project ownership
 
-despite their **Project Admin** access to the project.
+You can transfer your plan ownership to a different [organization](https://docs.platform.sh/administration/organizations.html) anytime.
 
-This enhaces consistency over your source code, as the Platform.sh project functions as a read-only mirror of your remote repository. Otherwise, changes pushed directly to the project would be overwritten or deleted when commits are pushed via the integration. Platform.sh considers your integrated remote repository to be the "source of truth" as soon as it has been configured, and this caveat ensures that all commits go through the integration.
+#### Before you begin
 
-The best course of action is to have your access updated on the integrated repository. If for some reason that is not a quick change, you can still clone through the project using the legacy pattern (which will set the *project* as its remote), but again, it is not recommended that you commit to the project once you have done so:
+You have to be the organization owner or an organization user with the **manage plans** permission.
 
-```bash
-$ git clone <project>@git.<region>.platform.sh:<project>.git
-```
+1. Invite the new organization owner as a **Project Admin**.
+2. Submit a [support ticket](https://console.platform.sh/-/users/~/tickets) from your organization owner's account to ask for the transfer.
+
+Once the transfer is completed, the new organization can administer all project settings, billing, and gets the subscription charges.
 
-## Transfer ownership
 
-If you're the owner of a project, and want to transfer it to a different user, first invite that user as a project administrator and then submit a support ticket from your owner account to ask for the transfer.
+##Troubleshooting
 
-Transfering a project automatically transfers the upcoming subscription charges to the new owner.
+[Source Integrations](integrations/source/troubleshooting.html)
```

Comment:
This link has no context that would make it useful. I'd put in something like "If you have X problem, see Y" or whatever. Also, the link format isn't right.